nifi-nar-bundles/nifi-social-media-bundle/nifi-twitter-processors/src/main/java/org/apache/nifi/processors/twitter/ConsumeTwitter.java

```diff
@@ -0,0 +1,366 @@
+    public static final PropertyDescriptor BASE_PATH = new PropertyDescriptor.Builder()
+            .name("base-path")
+            .displayName("Base Path")
+            .description("Specifies which base path the API client will use for HTTP requests. " +
+                    "Generally should not be changed from the default https://api.twitter.com except for testing")
+            .required(true)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .defaultValue("https://api.twitter.com")
+            .build();
+    public static final PropertyDescriptor BEARER_TOKEN = new PropertyDescriptor.Builder()
+            .name("bearer-token")
+            .displayName("Bearer Token")
+            .description("The Bearer Token provided by Twitter.")
+            .required(true)
+            .sensitive(true)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
+    public static final PropertyDescriptor QUEUE_SIZE = new PropertyDescriptor.Builder()
+            .name("queue-size")
+            .displayName("Queue Size")
+            .description("Maximum size of internal queue for streamed messages")
+            .required(true)
+            .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
+            .defaultValue("10000")
+            .build();
+    public static final PropertyDescriptor BATCH_SIZE = new PropertyDescriptor.Builder()
+            .name("batch-size")
+            .displayName("Batch Size")
+            .description("The maximum size of the number of tweets to be written to a single FlowFile." +
+                    "Will write less tweets if it there are not any tweets left in queue")
+            .required(true)
+            .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
+            .defaultValue("10")
+            .build();
+    public static final PropertyDescriptor BACKOFF_ATTEMPTS = new PropertyDescriptor.Builder()
+            .name("backoff-attempts")
+            .displayName("Backoff Attempts")
+            .description("The number of reconnection tries the processor will attempt in the event of " +
+                    "a disconnection of the stream for any reason, before throwing an exception. To start a stream after " +
+                    "this exception occur and the connection is fixed, please stop and restart the processor. If the value" +
+                    "of this property is 0, then backoff will never occur and the processor will always need to be restarted" +
+                    "if the stream fails.")
+            .required(true)
+            .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
+            .defaultValue("5")
+            .build();
+    public static final PropertyDescriptor BACKOFF_TIME = new PropertyDescriptor.Builder()
+            .name("backoff-time")
+            .displayName("Backoff Time")
+            .description("The duration to backoff to start attempting a new stream if" +
```

Comment:
Additional space needed:
```suggestion
            .description("The duration to backoff before requesting a new stream if " +
```

Reply:
Making the changes